aws/cerberus/cerberus.tf

```diff
@@ -0,0 +1,582 @@
+}
+
+variable "enable_https" {
+  description = "Adds infrastructure nessesary for https"
+  default     = "true"
+}
+
+variable "https_health_check_interval" {
+  description = "Average interval for https health check. Allowed Values:5-300"
+  default     = "300"
+}
+
+variable "enable_sftp" {
+  description = "Adds infrastructure nessesary for sftp"
+  default     = "true"
+}
+
+variable "sftp_health_check_interval" {
+  description = "Average interval for sftp health check. Allowed Values:10, 30"
+  default     = "30"
+}
+
+variable "parent_domain_name" {}
+
+locals {
+  module_kms_key_tags = "${merge(local.tags, map(
+    "TerraformModule", "cloudposse/terraform-aws-kms-key",
+    "TerraformModuleVersion", "0.1.2"))}"
+
+  remote_state_acct_key = "${coalesce(var.terraform_remote_state_acct_key, "master/${var.stage}/acct")}"
+  remote_state_vpc_key  = "${coalesce(var.terraform_remote_state_vpc_key, "master/${var.stage}/shared-vpc")}"
+}
+
+data "terraform_remote_state" "acct" {
+  backend = "s3"
+
+  config {
+    region         = "${var.aws_region}"
+    bucket         = "${var.namespace}-master-prd-tf-state-${var.master_account_id}"
+    encrypt        = true
+    key            = "${local.remote_state_acct_key}/terraform.tfstate"
+    dynamodb_table = "${var.namespace}-master-prd-tf-state-lock"
+    role_arn       = "arn:aws:iam::${var.master_account_id}:role/${var.master_account_assume_role_name}"
+  }
+}
+
+data "terraform_remote_state" "vpc" {
+  backend = "s3"
+
+  config {
+    region         = "${var.aws_region}"
+    bucket         = "${var.namespace}-master-prd-tf-state-${var.master_account_id}"
+    encrypt        = true
+    key            = "${local.remote_state_vpc_key}/terraform.tfstate"
+    dynamodb_table = "${var.namespace}-master-prd-tf-state-lock"
+    role_arn       = "arn:aws:iam::${var.master_account_id}:role/${var.master_account_assume_role_name}"
+  }
+}
+
+# ----------------------------------------------------------------------------------------------------------------------
+# MODULES / RESOURCES
+# ----------------------------------------------------------------------------------------------------------------------
+
+module "cerberus_kms_key" {
+  source                  = "git::https://github.com/cloudposse/terraform-aws-kms-key.git?ref=0.1.2"
+  namespace               = ""
+  stage                   = ""
+  name                    = "${local.stage_prefix}-cerberus"
+  description             = "${join(" ", list(var.desc_prefix, "KMS key for cerberus"))}"
+  deletion_window_in_days = 10
+  enable_key_rotation     = "true"
+  alias                   = "alias/${replace(local.stage_prefix, "-", "/")}/cerberus"
+  tags                    = "${local.module_kms_key_tags}"
+}
+
+resource "aws_security_group" "cerberus_ec2" {
+  count       = "${var.create == "true" ? 1 : 0 }"
+  name        = "${local.module_prefix}-ec2"
+  description = "${var.desc_prefix} Allow  traffic from internal and VPN and ALB"
```

Comment:
Description is `${var.desc_prefix} Allow traffic to Cerberus EC2 application instances`